.travis.yml

```diff
@@ -0,0 +1,35 @@
+sudo: required
+
+env:
+  global:
+    - ORACLE_COOKIE=sqldev
+    - ORACLE_FILE=oracle11g/xe/oracle-xe-11.2.0-1.0.x86_64.rpm.zip
+    - ORACLE_HOME=/u01/app/oracle/product/11.2.0/xe
+    - NLS_LANG=AMERICAN_AMERICA.AL32UTF8
+    - ORACLE_BASE=/u01/app/oracle
+    - LD_LIBRARY_PATH=$ORACLE_HOME/lib
+    - PATH=$PATH:$ORACLE_HOME/jdbc/lib
+    - DATABASE_VERSION=11.2.0.2
+    - ORACLE_SID=XE
+    - DATABASE_NAME=XE
+    - ORA_SDTZ='Europe/London' #Needed as a client parameter
+    - TZ='Europe/London'       #Needed as a DB Server parameter
+
+before_install:
+  - chmod +x .travis/oracle/download.sh
+  - chmod +x .travis/oracle/install.sh
+  - chmod +x .travis/setup_accounts.sh
+
+install:
+  - .travis/oracle/download.sh
+  - .travis/oracle/install.sh
+  - .travis/setup_accounts.sh
+  - bundle install
+
+language: ruby
+rvm:
+  - 2.3.0
+  - 2.2.4
+  - 1.9.3
+  - jruby-1.7.9
+  - jruby-9.0.4.0
```

Comment:
Sure, but I will not make it today. Need some good sleep after few too
short nights.
On 3 Mar 2016 17:40, "Yavor Nikolov" <notifications@github.com> wrote:

> In .travis.yml
> <https://github.com/rsim/ruby-plsql-spec/pull/33#discussion_r54916836>:
>
> > +  - chmod +x .travis/oracle/install.sh
> > +  - chmod +x .travis/setup_accounts.sh
> > +
> > +install:
> > +  - .travis/oracle/download.sh
> > +  - .travis/oracle/install.sh
> > +  - .travis/setup_accounts.sh
> > +  - bundle install
> > +
> > +language: ruby
> > +rvm:
> > +  - 2.3.0
> > +  - 2.2.4
> > +  - 1.9.3
> > +  - jruby-1.7.9
> > +  - jruby-9.0.4.0
>
> Can we try with jruby-9.0.5.0 instead of *4*?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rsim/ruby-plsql-spec/pull/33/files#r54916836>.
>